### DISCUSSION TITLE
Morphine tutorial_Age dependent PBPK

### DISCUSSION THREAD

**Discussion by xixi-1995** [Open-Systems-Pharmacology/Forum#826] [Questions & Problems] [answered]
Hello, everyone. I have a question for age dependent PBPK. In the tutorial, three pediatric groups were set as: 0-0.1 years (newborn), 0.1-2 years (toddlers), and 2-6 years (preschool).  I think this calculates postnatal age, but how do we take postmenstrual age into consideration? Postmenstrual age is important to age dependent maturation especially for younger children. Thank you.

**Comment by JanSchlender** [comment, marked as answer]
Hi Xixi,
Regular populations in PK-Sim consider Neonates as Term-born (PMA=40weeks). In case PMA is important for your task, you should select the Preterm population and define the PMA as well as PNA. In addition you can consider the aging processes during the simulation in case of longer timespans by selecting "allow aging" in the first tab of the "create simulation" screen.
Best,
Jan

**Comment by xixi-1995** [comment]
@JanSchlender Thank you so much. In addition, I want to know if the default ontogeny function for protein plasma and glomerular/tubular clearance is based on postnatal age, as I already know that enzymes ontogeny calculated the postmenstrual age.
